homeassistant/components/tellduslive/sensor.py

```diff
@@ -85,6 +132,11 @@ def _type(self):
         """Return the type of the sensor."""
         return self._id[1]
 
+    @cached_property
+    def entity_description(self) -> SensorEntityDescription:  # type: ignore[override]
+        """Return entity description for type."""
+        return SENSOR_TYPES[self._type]
```

Comment:
Do we need to guard here against `self_type` not being an item of `SENSOR_TYPES`, the old code did that.

Reply:
Good point! I'll change that.